src/utils.ts

```diff
@@ -1019,3 +1019,19 @@ export async function checkCompatibility(supabase: SupabaseClient<Database>, app
     localDependencies: dependenciesObject,
   }
 }
+
+// Function to get the remote checksum
+export async function getRemoteChecksum(supabase: SupabaseClient<Database>, appId: string, channel: string) {
```

Comment:
This cannot work, you have the data model in Supabase types.
You need to test locally your code by running our backend or create an account in prod.